tests/win_sec_attr.c

```diff
@@ -0,0 +1,96 @@
+/*
+    Copyright (c) 2012 Martin Sustrik  All rights reserved.
```

Comment:
Oh yeah probably a copy paste from another test case. I'll go over your code review comments and clean this up further sometimes in the next few days.

This code is only added to the cmake project files, so it should only be compiled and tested on Windows.